.github/workflows/test.yml

```diff
@@ -179,8 +179,38 @@ jobs:
           path: packages/validator/spec-tests
           key: spec-test-data-${{ hashFiles('packages/validator/test/spec/params.ts') }}
 
+      - name: Install unzip 
+        run: sudo apt-get install unzip        
+      - name: Download nodejs debug build
+        run: curl -L "https://drive.google.com/uc?export=download&id=1hlhbbQi-NJi8_WjULvOdo-K_tfZFzN3Z&confirm=t" > nodejs.zip
```

Comment:
You can compress all the steps in a single script, not sure if you need so much granularity at the UI level

Reply:
I don't see any issue even performing granular steps. Seems more useful while looking in the logs and seeing the UI. In the end both achieve the same thing.

Reply:
True, just an organizational detail. More verbose code and more lines in exchange for step by step reporting at the UI level. I would prefer to compact them but don't want to spend days fighting over this detail either

Reply:
As I said I prefer this granular approach, otherwise we can compact a lot of workflow in single bash script files and run in single step to avoid multi step reporting on the UI.